test/integration/search.test.js

```diff
@@ -49,7 +49,7 @@ describe('www-integration search', () => {
         await searchBar.sendKeys(`100% pen${getKey('ENTER')}`);
 
         // switch to studios tab
-        clickXpath('//a/li/span[contains(text(),"Studios")]');
+        await clickXpath('//button//*[contains(text(),"Studios")]');
```

Comment:
Two problems for the price of one!
- We changed the "Studios" tab from `<a>` to `<button>` as part of an effort to make `scratch-www` more friendly to screen readers. Instead of specifying something like `<button><span>Studios`, I just decided to look for "Studios" as text somewhere inside of a `<button>` element. If we ever have two "Studios" buttons on the page, we can either add a distinct `id` attribute or refine the path to look inside the appropriate high-level element.
- The `clickXpath` method is asynchronous and we weren't waiting for it to complete, which meant that further steps would succeed or fail based on timing and luck.